scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -152,6 +154,7 @@ package object dialects {
     .withAllowStarWildcardImport(true)
     .withAllowProcedureSyntax(false)
     .withAllowDoWhile(false)
+    .withAllowPlusMinusUnderscoreIdent(true)
```

Comment:
should we start creating additional scala3 dialects? if we have lots of scala2.1x, why not scala3.1 and scala3.2?

Reply:
We will most likely need to, but currently I am not sure what will be in 3.1 at this point.

Reply:
we know that this flag we are adding is only in 3.2, right? at least according to the docs.

Reply:
@kitbellew The new syntax is already available under a flag `-Ykind-projector:underscores` in 3.
See
* https://docs.scala-lang.org/scala3/guides/migration/plugin-kind-projector.html
* https://dotty.epfl.ch/docs/reference/changed-features/wildcards.html